aa-modules-upgrade-bug-RN.md (#2459)
RedisLabs/redislabs-docs: content/rs/release-notes/rs-6-2-18.md

- Compatibility with open source Redis v6.2.6
- Additional enhancements and fixes

### Known upgrade limitation

> Do not upgrade a cluster that hosts Active-Active databases with module/s before following the next steps:

Reviewer: ```suggestion
Before you upgrade a cluster that hosts Active-Active databases with modules, ensure you complete these steps:
```

> Check the module and its version as it appears in the db config and in the default db config using the `crdb-cli crdb get --crdb-guid <crdb-guid>`. The fields are called 'modules' and 'module_list'.

Reviewer: ```suggestion
1. Use the `crdb-cli crdb get --crdb-guid <crdb-guid>` command to check that the modules (`modules`) and their versions (`module_list`) are as they appear in the db config and in the default db config. 
```

> Via the UI Admin console, under the 'redis modules' tab, validate that these module/s with their specific version/s are loaded to the cluster.

Reviewer: ```suggestion
1. From the admin console's **Redis Modules** tab, validate that these modules with their specific versions are loaded to the cluster.
```

> In case one or more of the modules+versions are missing, or if you need help [contact Redis support](https://redis.com/company/support/) before taking additional steps.

Reviewer: ```suggestion
If one or more of the modules/versions are missing, or if you need help, please [contact Redis support](https://redis.com/company/support/) before taking additional steps.
```
